src/plugins/synced-lyrics/renderer/components/LyricsContainer.tsx

```diff
@@ -48,29 +57,43 @@ export const LyricsContainer = () => {
 
       <Switch>
         <Match when={!lineLyrics().length}>
-          <yt-formatted-string
-            class="warning-lyrics description ytmusic-description-shelf-renderer"
-            text={{
-              runs: [
-                {
-                  text: 'No lyrics found for this song.',
+          <Show when={isInstrumental()}>
```

Comment:
Use `Match` instead of duplicate `Show`

Reply:
I did that originally, but I decided the code was too ugly, so I didn't keep it.
Is it really necessary?